src/routes.ts

```diff
@@ -9,9 +9,11 @@ export const HOME = '/',
     TRANSLATION_ADD = '/term/:termId/translation/add',
     TRANSLATION = '/term/:termId/translation/:translationId',
     TRANSLATION_EXAMPLE = '/term/:termId/translation/:translationId/translation-example/:translationExampleId',
+    TRANSLATION_EXAMPLE_REDIRECT = '/translation-example/:translationExampleId',
     TRANSLATION_EXAMPLE_ADD = '/term/:termId/translation/:translationId/translation-example/add',
     USER = '/user/:userId',
     ADMIN = '/admin',
+    ADMIN_CONTENT = '/adminContent',
```

Comment:
So far we didn't camelCase paths, so maybe:
```suggestion
    ADMIN_CONTENT = '/admin/content,
```